tools/boot/install-boot.sh

```diff
@@ -59,7 +59,12 @@ make_esp_file() {
     stagedir=$(mktemp -d /tmp/stand-test.XXXXXX)
     mkdir -p "${stagedir}/EFI/BOOT"
     efibootname=$(get_uefi_bootname)
+
     cp "${loader}" "${stagedir}/EFI/BOOT/${efibootname}.efi"
+    loader="${loader%.efi}_ia32.efi"
```

Comment:
I really don't like this magic... can we not just make it optionally take extra files as arguments and update the release scripts?

Reply:
Ah, I didn't think of that. That would be much better, I think.